fluent.runtime/docs/index.rst

```diff
@@ -0,0 +1,26 @@
+.. fluent.runtime documentation master file, created by
+   sphinx-quickstart on Thu Jan 24 15:45:23 2019.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to fluent.runtime's documentation!
+==========================================
+
+These are the docs for ``fluent.runtime`` |release|. Check the :doc:`/history` for
+significant changes.
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+   installation
+   usage
+   history
+
+
+Indices and tables
+==================
+
+* :ref:`genindex`
+* :ref:`modindex`
+* :ref:`search`
```

Comment:
What are these? They're either empty or 404 on https://fluent-runtime.readthedocs.io/en/latest/.

Reply:
Hmm, these were created automatically by Sphinx quick-start.

* `search` isn't needed - there is a search box in the sidebar on read the docs. 
* `genindex` is a general index, described [here](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html?highlight=genindex#special-names) in the sphinx docs. I'm guessing we're unlikely to use it.
* `modindex` is an index of modules/classes, which is currently empty because we have no actual API docs. I think API docs would be a good addition. But I'll remove it for now, I can add it back later.